### ISSUE TITLE
#7637 [a+] big django & wagtail update: Error-Symbol in "Dokument" Tab doesn't work correctly in "Text Diskutieren" Module

### ISSUE THREAD

**Issue by sblindow** [liqd/adhocracy-plus#2590]
**URL:**
You need to add a new "Text Diskutieren" Module in any project to recreate this issue.

**user:**
Initiators

**expected behaviour:**
When I enter Text in the "Dokument" Tab in the "Text Diskutieren" Module, the red exclamation mark should disappear after saving, to indicate that everything has been entered correctly.

**behaviour:**
Currently, the exclamation mark does not disappear after saving. Only when switching to another tab, for ex. "Phasen", does it vanish.

**important screensize:**
not relevant

**device & browser:**
MacOS, Firefox 125.0.2 (64-Bit)

**Screenshot?**

<img width="1396" alt="Bildschirmfoto 2024-04-24 um 13 01 54" src="https://github.com/liqd/adhocracy-plus/assets/144320958/4c1dcd65-f9fd-4e61-8e80-fcc1f00c2067">
"Dokument" Tab in fresh "Text Diskutieren" module

<img width="1648" alt="Bildschirmfoto 2024-04-24 um 13 02 42" src="https://github.com/liqd/adhocracy-plus/assets/144320958/521b3a7b-ea7a-40cf-bdeb-84712230696a">
Text has been entered.

 
<img width="1402" alt="Bildschirmfoto 2024-04-24 um 13 02 50" src="https://github.com/liqd/adhocracy-plus/assets/144320958/317acfe6-9f48-46fe-940b-97b6a925e03b">
After saving the changes, the red exclamation mark on the left does not disappear, although the criteria to publish the module are met.

<img width="1452" alt="Bildschirmfoto 2024-04-24 um 13 02 59" src="https://github.com/liqd/adhocracy-plus/assets/144320958/ba848ac2-97bd-434b-9c04-e92846f4ac62">
Only after switching to another tab, does it clear.

**Comment by TeaJayyy** [comment]
@hom3mad3 it has low priority!

**Comment by TeaJayyy** [comment]
@hom3mad3 Yes sure, thought it would be an easy fix :(

**Comment by goapunk** [comment]
@hom3mad3 @TeaJayyy This worked before I updated the documents for the new ckeditor and should be easy to fix (it just reloads the page if the save was successful). I can look into why it stopped working

**Comment by TeaJayyy** [comment]
@goapunk I love easy fixes!
